diff --git a/src/Misc/MiddleWare.cpp b/src/Misc/MiddleWare.cpp
index 4210c5ae..b7767f4f 100644
--- a/src/Misc/MiddleWare.cpp
+++ b/src/Misc/MiddleWare.cpp
@@ -598,9 +598,11 @@ public:
         Part *p = alloc.get();
 #else
         Part *p = new Part(*master->memory, synth, master->time,
+                master->sync,
                 config->cfg.GzipCompression,
                 config->cfg.Interpolation,
-                &master->microtonal, master->fft);
+                &master->microtonal, master->fft, &master->watcher,
+                ("/part"+to_s(npart)+"/").c_str());
         p->partno  = npart % NUM_MIDI_CHANNELS;
         p->Prcvchn = npart % NUM_MIDI_CHANNELS;
 
